paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/EnableInstantDebitsSettingsDefinition.kt

```diff
@@ -10,11 +9,8 @@ internal object EnableInstantDebitsSettingsDefinition : BooleanSettingsDefinitio
 ) {
 
     override fun configure(value: Boolean, checkoutRequestBuilder: CheckoutRequest.Builder) {
-        // Hijacking this method to synchronize the setting with the feature flag
-        FeatureFlags.instantDebits.setEnabled(value)
-    }
-
-    override fun valueUpdated(value: Boolean, playgroundSettings: PlaygroundSettings) {
-        FeatureFlags.instantDebits.setEnabled(value)
+        if (value) {
+            checkoutRequestBuilder.supportedPaymentMethods(listOf("card", "link"))
```

Comment:
I'm wondering if we should do this differently.

Should we add this as it's own SettingsDefinition? Otherwise this could clash with another SettingsDefinition, since we set this elsewhere too, right?

Reply:
I'm suggesting we remove `EnableInstantDebitsSettingsDefinition`, and add `SupportedPaymentMethodsSettingsDefinition` to `uiSettingDefinitions`.

Reply:
opened [this](https://github.com/stripe/stripe-android/pull/8428) based on Jay's suggestion. Also if multi choice is supported on PaymentSheet settings at some point we can migrate the comma separated string to some payment method picker ([this](https://github.com/stripe/stripe-android/blob/299f56e37b5f2a202c0133cf0dca1d0ee5c71e7d/financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/settings/Setting.kt#L35-L48) is how we do multichoice on Financial Connections)

Reply:
Rebased onto Carlos’ pull request!